pkg/review/commit_review_status.go

```diff
@@ -135,6 +142,14 @@ type CommitApprovalDoFn struct {
 	GithubClient *githubv4.Client
 }
 
+// BreakGlassIssueDoFn is an object that implements beams "DoFn" interface to
+// provide the processing logic for converting retrieving the associated break
+// glass issue for a CommitReviewStatus.
+type BreakGlassIssueDoFn struct {
```

Comment:
It's more common to name function types with the `Func` suffix (e.g. `context.CancelFunc`). However, this appears to be a struct.

Reply:
This struct is an Apache Beam DoFn style struct (https://beam.apache.org/documentation/programming-guide/#4212-creating-a-dofn) which is what the name is intending to convey.

Reply:
Ah TIL. Thanks!